apps/web/pages/team/[slug].tsx

```diff
@@ -277,12 +284,23 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   );
   const isOrgContext = isValidOrgDomain && currentOrgDomain;
 
+  // Provided by Rewrite from next.config.js
+  const isOrgProfile = context.query?.isOrgProfile === "1";
   const flags = await getFeatureFlagMap(prisma);
+  const isOrganizationFeatureEnabled = flags["organizations"];
+
+  log.debug("getServerSideProps", {
+    isOrgProfile,
+    isOrganizationFeatureEnabled,
+    isValidOrgDomain,
+    currentOrgDomain,
+  });
+
   const team = await getTeamWithMembers({
     slug: slugify(slug ?? ""),
     orgSlug: currentOrgDomain,
     isTeamView: true,
-    isOrgView: isValidOrgDomain && context.resolvedUrl === "/",
```

Comment:
resolvedUrl is what we get after rewrite, so this should never be / AFAIU.
I am using a more reliable approach of passing the query param itself when we do the rewrite, because that's the logic that decides it's an org link and redirects accordingly.